docs/requirements.txt

```diff
@@ -12,4 +17,3 @@ sklearn
 pytest
 pytest-xdist
 # Must install jax itself for notebook execution to work
-.
```

Comment:
I belive that removing ``.`` here is what caused the CI errors (`opt_einsum` is listed as a jax dependency, and if you don't install jax from the current directory you don't install its dependencies)

Reply:
Oops, I didn't even see in `git diff` that I'd removed that!  Force-pushed a fix.